sys-apps/sum-driver/sum-driver-1.6.0.20151201.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eapi7-ver linux-mod
+
+MY_DATE="$(ver_cut 4)"
+MY_PN="${PN//-/_}"
+MY_PV="$(ver_cut 1-3)"
+
+DESCRIPTION="Supermicro Update Manager (SUM) kernel module"
+HOMEPAGE="https://www.supermicro.com"
+SRC_URI="${MY_PN}_V${MY_PV}_${MY_DATE}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="supermicro"
+SLOT="0"
+
+RESTRICT="bindist fetch mirror strip"
```

Comment:
Why the strip restriction?

Reply:
I set this because of the supermicro license. Or does strip do nothing here?

Reply:
You mean because the EULA prohibits 'modifying' the 'software'? @gentoo/licenses, do we block stripping for that? Does it actually work?

Reply:
> You mean because the EULA prohibits 'modifying' the 'software'?

Yes.

Reply:
Installation is explicitly allowed, and I would see stripping as part of the installation process, not as modification.

Disclaimer: IANAL, TINLA.

Reply:
Okay, thank you. Removed `strip`.